### DISCUSSION TITLE
Potential issues with midi's, probably nothing.

### DISCUSSION THREAD

**Discussion by SomeoneElsHere** [space-wizards/space-station-14#26697] [Questions] [answered]
Is there a size limit on how big of a file the game can process?  How much of an impact is file size on the game?

A potential way to slow down the game for others could be to create a very large midi by file formatting another file, then trying to play the 'song' on the midi loader. I don't know quite how the midi system works so I'm just wondering if it could be an issue.

Thanks!

**Comment by PJB3005** [comment, marked as answer]
MIDI files are streamed from the source client, large MIDI files are not uploaded directly. Actual MIDI event rate is limited so you can't just play black MIDIs with too many notes either.

Regardless, there is a file size limit.
